config/samples/secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ntnx3-ndb-secret
+type: Opaque
+stringData:
+  username: ntnx3
+  password: ntnx3
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ntnx3-db-secret
+type: Opaque
+stringData:
+  password: NTNX3
+  ssh_public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCslQJ1257QMu3TES1R3m8vYkRHGywcuCBq1c3RRMbAwe7nNbmWpcMwEoXhCXCqxxo12XCM59svv7XvoLPnCAUMxQap1BjEbRwP4BbFIbK+OLzJ+22j5FLj4fAGJx62s6BvG1+QeRCoW96C+qR92FP6f8wNmLfiskealYR5UsXaM8HOdOhCkVvm4C1A9L0V+Vr257TvW84mR9rxNws4xqSevkS51yq5IW4h+JmbkWLnFcbnGhaywKUOrS0X8wwv+XtxIslf+OQk1PAcvX5tsQ1b70qcrpp+ZxtmMHTOZdsnUqeFNb7AYa7PoR7hHbLxcBZwqXA61OeLsydScqHQw9oigpFf9PbZyd9dN8E/TnCRrxYzJtPUNgAnTyLBfDFCh/b/z3GBdyScDXJhsM8Wa0XXMqbYZFxsDO6PQByQIC+Bf7QYsJJDS3j+SuqMX1FVAjFrC4lpbJ+EsbLjT3tBWKDltg7N7BGpZ9qJ3PlekqPaNWmCvYX++P/13tiodfiyrE8GmtcT4sqFsYrPxT/mHL/nEE/NaEXpRueZUkSlPG/BwpG3OtM3TWrXHK7xzQ0KLp0gP5NqTaCnKH9oqH5ZrVnkCVwpbDZ62GAsThs4S3HyhSxAxEZdrYwUU2+Mmjo+psVgCadkKSUWhbTzL4fqHfF67e5u6xv/iCQbr1F+feoF8Q== work@Vishnus-MacBook-Pro.local
+
```

Comment:
Please remove this file from your check-in.

Reply:
Yes, I'll be reverting these check ins